src/tools/pkcs11-tool.c

```diff
@@ -2919,8 +2918,13 @@ static int gen_keypair(CK_SLOT_ID slot, CK_SESSION_HANDLE session,
 				if (!strcmp(ec_curve_infos[ii].oid, type + 3))
 					break;
 			}
-			if (!ec_curve_infos[ii].name)
-				util_fatal("Unknown EC key params '%s'", type + 3);
+			if (!ec_curve_infos[ii].name) {
+				fprintf(stderr, "EC key parameters may be specified by their canonic name or object identifier. Possible values are:\n");
+				for (ii=0; ec_curve_infos[ii].name; ii++)   {
```

Comment:
Can you fix the spacing here as suggested by the clang-format:
```
diff --git a/src/tools/pkcs11-tool.c b/src/tools/pkcs11-tool.c
index 8478f41..9c68081 100644
--- a/src/tools/pkcs11-tool.c
+++ b/src/tools/pkcs11-tool.c
@@ -2920,7 +2920,7 @@ static int gen_keypair(CK_SLOT_ID slot, CK_SESSION_HANDLE session,
 			}
 			if (!ec_curve_infos[ii].name) {
 				fprintf(stderr, "EC key parameters may be specified by their canonic name or object identifier. Possible values are:\n");
-				for (ii=0; ec_curve_infos[ii].name; ii++)   {
+				for (ii = 0; ec_curve_infos[ii].name; ii++) {
 					fprintf(stderr, "%s (%s)\n", ec_curve_infos[ii].name, ec_curve_infos[ii].oid);
 				}
 				util_fatal("Unknown EC key parameter '%s'", type + 3);
```

Reply:
@frankmorgner can you squash the style issue so we can merge this?

Reply:
done and rebased on top of main branch